src/assign.ts

```diff
@@ -153,7 +152,8 @@ export const selectReviewers = async (
     const randomFileOwner = randomize(stack.find(owner => owner.pattern === randomFile)?.owners)?.shift()
     debug(`Selected random file owner: ${randomFileOwner}`)
     const randomGlobalCodeowners = randomize(globalCodeowners)
-    const selected = randomFileOwner ?? randomGlobalCodeowner(randomGlobalCodeowners)
+    const selected = randomFileOwner ?? randomGlobalCodeowners?.[0]
+    if (selected && !isTeam(selected)) randomGlobalCodeowners?.shift()
```

Comment:
`selected` being `undefined | null` is already handled in the following two checks and we already `break` if `!selected`, so this check is redundant 😄 

```suggestion
    if (!isTeam(selected)) randomGlobalCodeowners?.shift()
```

Reply:
I'd rather we change `isTeam` to handle a `string?` parameter 😄 to keep the logic consistent